content/cosign-2-0-released.md

```diff
@@ -0,0 +1,44 @@
+* Signing OCI images by tag prints a warning and is strongly discouraged, e.g. `cosign sign container.registry.io/foo:tag`. This is considered insecure since tags are mutable. If you want to specify a particular image, you are recommended to do so by digest.
+* SCT verification, a proof of inclusion in a certificate transparency log, is now on by default for verifying Fulcio certificates. For private deployments without certificate transparency, use `--insecure-ignore-sct=true` to skip this check.
+* DSSE support in `verify-blob` has been removed. You can now verify attestations using `verify-blob-attestation`.
+* Environment variable `SIGSTORE_TRUST_REKOR_API_PUBLIC_KEY` has been removed. For private deployments, if you would like to set the Rekor public key to verify transparency log entries, use either a [TUF setup](https://blog.sigstore.dev/sigstore-bring-your-own-stuf-with-tuf-40febfd2badd) or set `SIGSTORE_REKOR_PUBLIC_KEY` with the PEM of the custom Rekor public key..
+* `verify-blob` no longer searches for a certificate. You must provide one with either `--certificate` or `--bundle`.
+* `cosign attest --type {custom|vuln}` (and `cosign verify-attestation`) will now use the RFC 3986 compliant URIs, adding `https://`, so that these predicate types are compliant with the in-toto specification.
+* The `CosignPredicate` envelope that wraps the predicates of SPDX and CycloneDX attestations has been removed, which was a violation of the schema specified via the `predicateType` field ([more information](https://github.com/sigstore/cosign/pull/2718)).
+
+Additionally, we've made the following improvements:
+
+* Blob attestation and verification is now supported with `cosign attest-blob` and `cosign verify-blob-attestation`.
+* You can now set flags via environment variables, for example instead of `--certificate-identity=email`, you can set an environment variable for `CERTIFICATE_IDENTITY=email`.
+* `--offline=true` removes the fallback to the Rekor log when verifying an artifact. Previously, if you did not provide a bundle (a persisted response from Rekor), Cosign would fallback to querying Rekor. You can now skip this fallback for offline environments. Note that if the bundle fails to verify, Cosign will not fallback and will fail early.
+* A Fulcio certificate can now be issued for self-managed keys by providing `--issue-certificate=true` with a key, `--key`, or security key, `--sk`. This is useful when [adopting Sigstore incrementally](https://blog.sigstore.dev/adopting-sigstore-incrementally-1b56a69b8c15/).
+* Experimental support for trusted timestamping has been added. We will be writing more about this in an upcoming blog post!
```

Comment:
@haydentherapper shouldn't we mention how to verify images against a Sigstore or a non-TSA  instance ?

Reply:
My plan was to have a dedicated blog post for that, but if you think it'd be useful, I can add in example commands with the new flags.

Reply:
I feel we must mention it, we can then write a blog post explaining it in more details.

Reply:
In my opinion, it is an important feature that especially opens the trust model.

Reply:
Added!